apps/client/src/root/root.tsx

```diff
@@ -0,0 +1,23 @@
+import { Outlet, useNavigate } from 'react-router-dom';
+import { Page } from '../page/page';
+import { usePageType } from '../page/page-type.hook';
+import { usePageLocation } from '../page/page-location.hook';
+
+export interface AppProps {
+  signOut?: () => void;
+}
+
+export const Root = () => {
+  const navigate = useNavigate();
+  const pageType = usePageType();
+  const location = usePageLocation();
+
+  return (
+    <Page
+      content={<Outlet />}
+      pageType={pageType}
+      location={location}
+      setLocation={navigate}
+    />
+  );
+};
```

Comment:
this is clean 👏

Reply:
Trying my best to make the lower-level components stateless and without dependencies that will make them difficult to test!